.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  root: true,
+  env: {
+    es2020: true,
+    node: true,
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2020,
+  },
+  plugins: ["@typescript-eslint", "prettier"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier/@typescript-eslint",
+  ],
+  ignorePatterns: [
+    "abis/",
+    "build/",
+    "generated/",
```

Comment:
I would tailor these `ignorePatterns` to the OBA project, these were `dex-subgraph` specific - I guess this depends on where Waffle + buidler place its intermediate files.

Reply:
There is a build folder, I have removed `generated` and `abis`